Source/App/CMakeLists.txt

```diff
@@ -45,4 +46,4 @@ if (MSVC OR MSYS OR MINGW OR WIN32)
                            SvtHevcEnc)
 endif()
 
-install(TARGETS SvtHevcEncApp RUNTIME DESTINATION bin)
```

Comment:
Could you potentially remove this change? This will conflict with the cmakelists PR and prevent easy patching?